scripts/docs.angularjs.org-firebase/readme.firebase.docs.md

```diff
@@ -13,9 +13,13 @@ See .circleci/config.yml for the complete deployment config and build steps.
 
 # Serving locally:
 
+- Run `grunt:package`.
+  This builds the files that will be deployed.
+
```

Comment:
I removed the `package` step from `prepareDeploy` because on the CI it has already run.